src/seismometer/_api.py

```diff
@@ -176,86 +183,126 @@ def fairness_audit(metric_list: Optional[list[str]] = None, fairness_threshold=1
     sensitive_groups = sg.cohort_cols
     metric_list = metric_list or ["tpr", "fpr", "pprev"]
 
-    df = pdh.event_score(
-        sg.data(),
-        sg.entity_keys,
-        score=sg.output,
-        ref_event=sg.predict_time,
-        aggregation_method=sg.event_aggregation_method(sg.target),
-    )[[sg.target, sg.output] + sensitive_groups]
-
-    display_fairness_audit(
-        df, sensitive_groups, sg.output, sg.target, sg.thresholds[0], metric_list, fairness_threshold
+    return generate_fairness_audit(
+        sensitive_groups,
+        sg.target,
+        sg.output,
+        sg.thresholds[0],
```

Comment:
Q: I see that thresholds is a list of floats. What is the significance of thresholds[0]?

Reply:
its the first threshold in metadata.json, so potentially the most important? Since the list is unordered, its kind of hard to tell, but in most cases we have just said 1st = primary. This is why the explore controls allow you to change this.

Reply:
Its currently used in the two older plotting functions (`plot_leadtime_enc` and `fairness_audit`), which will be removed in a future PR because we now have the Explore* controls that will replace them